config/src/test/java/org/springframework/security/config/web/server/HeaderSpecTests.java

```diff
@@ -49,7 +50,7 @@
  */
 public class HeaderSpecTests {
 
-	private ServerHttpSecurity.HeaderSpec headers = ServerHttpSecurity.http().headers();
+	private ServerHttpSecurity serverHttpSecurity = ServerHttpSecurity.http();
```

Comment:
To be consistent with the variable we typically use, please change to `http`. We also use http to make it look [similar to the XML based configuration](https://spring.io/blog/2013/07/11/spring-security-java-config-preview-readability/#comparison-to-xml-namespace).